src/fundus/publishers/base_objects.py

```diff
@@ -75,8 +75,8 @@ def supports(self, source_types: List[Type[URLSource]]) -> bool:
         for source_type in source_types:
             if not inspect.isclass(source_type) or not issubclass(source_type, URLSource):
                 raise TypeError(
-                    f"Got unexpected type '{source_type}'. "
-                    f"Allowed are '{', '.join(cls.__name__ for cls in iterate_all_subclasses(URLSource))}'"
+                    f"Got unexpected type {source_type!r}. "
+                    f"Allowed are {', '.join(repr(cls.__name__) for cls in iterate_all_subclasses(URLSource))}"
```

Comment:
If there is no special reason to do it this way, I would suggest, for consistencies sake, to do it this way: `', '.join(cls.__name__ for ... )!r`